shared_python/FinalTables.py

```diff
@@ -24,7 +24,7 @@ def original_table(self, table_name, filter='', database_name=None):
         return self.sql.execute_dict(query)
 
     def _escape_unescape(self, item):
-        return html.unescape(item).replace('\\', '\\\\').replace('"', '\\"').replace("'", "\\'")
+        return html.unescape(item).replace('\\', '\\\\').replace('"', '\\"').replace("'", "\\'").replace("%", "%%")
```

Comment:
This looks good, though two questions:
1. did you double-check that the `%` appears correctly in the MySQL database?
2. could you write a test for this? I realise you'll have to create a new test file in /test but we do have a couple already that you can copy and I think it will be really worth it in the long run.

Reply:
yes, I've checked that the percent symbols appear correctly in the database. Sure! I'll write a test for this:)